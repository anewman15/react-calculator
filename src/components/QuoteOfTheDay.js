/*
 eslint-disable no-unused-vars
*/

import React from 'react';
import Button from 'react-bootstrap/Button';

const QuoteOfTheDay = () => (
  <div className="w-75 mx-auto my-5">
    <div className="card">
      <div className="card-header display-4 font-weight-bold">
        Quote
      </div>
      <div className="card-body p-5">
        <blockquote className="blockquote mb-0">
          <p>The beauty of mathematics only shows itself to more patient followers</p>
          <footer className="blockquote-footer">Maryam Mirzakhani</footer>
        </blockquote>
      </div>
    </div>
    <div className="text-center my-4">
      <Button className="w-50 text-white" variant="success" href="/calculator">Let's Go Do Math</Button>
    </div>
  </div>
);

export default QuoteOfTheDay;