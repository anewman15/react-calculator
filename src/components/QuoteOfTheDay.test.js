/*
 eslint-disable no-unused-vars, import/named, import/no-cycle
*/

import React from 'react';
import { act, render } from '@testing-library/react';
import { unmountComponentAtNode } from 'react-dom';
import pretty from 'pretty';
import QuoteOfTheDay from './QuoteOfTheDay';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('QuoteOfTheDay', () => {
  it('should render a page with a quote', () => {
    act(() => {
      render(<QuoteOfTheDay />, container);

      expect(pretty(container.innerHTML))
        .toMatchSnapshot();
    });
  });
});
