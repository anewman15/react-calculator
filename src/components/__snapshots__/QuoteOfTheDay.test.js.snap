// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteOfTheDay should render a page with a quote 1`] = `""`;
